pkg/core/blockchain.go

```diff
@@ -504,6 +506,11 @@ func (bc *Blockchain) persist() error {
 			"blockHeight":     bHeight,
 			"took":            time.Since(start),
 		}).Info("blockchain persist completed")
+
+		// update monitoring metrics.
+		updateBlockHeightMetric(bHeight)
```

Comment:
I don't think it makes sense changing this one here, it's only changed in the `storeBlock()`.

Reply:
removed